### PR TITLE
Added basic UI

### PR DESCRIPTION
**Pull request by Arch-Noize**
Now that my program has been given every function for the library, there's a complete new UI that allows a user to:
- Add a person (student or teacher)
- Add books
- Rent books
- List all books in the library
- List all people registered in the library
- And list books rented by someone

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    if @books.empty?
+      puts 'Currently, there are no books!'
+    else
+      @books.each_with_index do |book, i|
+        puts "#{i}) Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
```
Comment (gemmen29) [inline app.rb:28]: - [ ] Kindly fix this problem in the `list all people` option for displaying the `Teacher` as for teacher age display the name and for teacher name display age.
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    puts 'Title:'
+    title = gets.chomp.to_s
+
+    puts 'Author:'
+    author = gets.chomp.to_s
+
+    @books.push(Book.new(title, author))
+  end
+
+  def add_rental
```
Comment (gemmen29) [inline app.rb:101]: - [ ] Kindly fix this error in the `Create a rental` option

image

When I try to create a new rental I get this error.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Arch-Noize, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Arch-Noize , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- **Awesome job, Ruido,** fixing the issues. Your app is now working correctly . 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**